src/main/java/com/epam/lab/paymentsystem/controller/OperationController.java

```diff
@@ -86,6 +90,12 @@ public String getUserHistory(
     return HISTORY_PAGE;
   }
 
+  @GetMapping(value = "/history")
+  public String getUserHistory2() {
```

Comment:
Also, I think it should be replaced to `IndexController` or new `TabBarController` since this functional is only proxy